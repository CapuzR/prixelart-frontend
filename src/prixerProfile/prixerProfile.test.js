import React from 'react';
import renderer from 'react-test-renderer';
import PrixerProfile from "./prixerProfile";

jest.mock('@material-ui/core/CssBaseline', () => 'CssBaseline')
jest.mock('../sharedComponents/appBar/appBar', () => 'AppBar')
jest.mock('./userData/userData', () => 'UserData')
jest.mock('./prixerOptions/prixerOptions', () => 'PrixerOptions')
jest.mock('./grid/grid', () => 'Grid')

it('<PrixerProfile /> renders correctly', () => {
  const tree = renderer
    .create(<PrixerProfile />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});