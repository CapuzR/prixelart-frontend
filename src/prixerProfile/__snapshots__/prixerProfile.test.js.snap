// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PrixerProfile /> renders correctly 1`] = `
<main
  className="MuiContainer-root MuiContainer-maxWidthXs"
>
  <CssBaseline />
  <AppBar />
  <UserData />
  <PrixerOptions />
  <Grid />
</main>
`;
